packages/fxa-settings/src/pages/Signin/Signin/index.tsx

```diff
@@ -0,0 +1,248 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import React, { ReactElement, useCallback, useState } from 'react';
+import { usePageViewEvent } from '../../../lib/metrics';
+import { useFtlMsgResolver } from '../../../models';
+import { FtlMsg } from 'fxa-react/lib/utils';
+import { RouteComponentProps } from '@reach/router';
+import InputPassword from '../../../components/InputPassword';
+import LinkExternal from 'fxa-react/components/LinkExternal';
+import { ReactComponent as PocketLogo } from './pocket.svg';
+
+type SigninProps = {
+  email: string;
+  isPasswordNeeded: boolean;
+  isPocketClient: boolean;
+  serviceLogo?: React.ElementType;
```

Comment:
Seems like this should be `ServiceLogo` given it's an element? Then you can remove the `const ServiceLogo = serviceLogo` below.